README.md

```diff
@@ -8,27 +10,56 @@ Education and communications around the value of Monero. This involves writing,
+
+Check out what we are working on: https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/26
+
+Our gitlab (backup incase this goes down): https://gitlab.com/monero-outreach/Contribute/blob/master/README.md
+
+## Translations
+
+Please review the translations tips (standardized terms) we expect everyone to follow: [Tips for Monero translators](https://github.com/monero-ecosystem/monero-translations/blob/master/translation-tips.md).
+
+For XMR Rewards visit [Taiga EPIC #69 Monero Outreach Localization](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/69).
+
+
+Follow this instructions (if you are not familiar with Github, check out this [tutorial](https://guides.github.com/activities/hello-world/)):
 
+1. Fork the repository and clone your fork locally.
+2. The directory `en` has all the latest documents.
+3. If you are working on a new language, add a `subfolder` for your language in the `translations` folder (use [country codes](https://wiki.openstreetmap.org/wiki/Nominatim/Country_Codes) for the `subfolders`). Copy the .md files you are going to translate from the `en` folder and paste it in the `subfolder` you just created. Use the correct folders for locating your work.
+4. **File Formats** - Documents are worked in .md format. Md stands for markdown, markdown is a lightweight markup language with plain text formatting syntax. It's really easy to use and you can use any text editor to add text. Need help about mastering markdown format? [Here](https://guides.github.com/features/mastering-markdown/) is a tutorial.
+ - Design edits will be done once you submit your work.
+5. When saving changes, remember to add a message to the commit with the **documents name, language, and a small description of your work** (e.g. **_"Quickfacts fr translation"_**, **_"Missing punctuation donttalk PSA pt_Br"_**, etc.).
+6. If you are a reviewer please visit, [Taiga EPIC #69 Monero Outreach Localization](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/69) to get assigned a translation.
+7. Happy contributing! 
+
+### Fonts
+
+Fonts can be found inside [monero-outreach-docs folder](https://github.com/monero-ecosystem/outreach-docs/tree/master/monero-outreach-docs/fonts/monero%20fonts) or directly in the site of Monero Outreach - [Monero Typography](https://www.monerooutreach.org/monero-typography.php). _Fonts are actually not needed while using .md file format, but you can use them if you are currently working on any Monero documents, these fonts have been edited by the Monero Outreach Team._
+
+## Resources
 - Web: [monerooutreach.org](https://www.monerooutreach.org/)
-- Mail: [monerooutreach@protonmail.com](mailto:monerooutreach@protonmail.com)
+- Mail: 
+   - [monerooutreach@protonmail.com](mailto:monerooutreach@protonmail.com)
+   - [paintluis@protonmail.com](mailto:paintluis@protonmail.com)
+   - [monerooutreach@protonmail.com](mailto:monerooutreach@protonmail.com)
```

Comment:
Hi @erciccione. I think this e-mail address might be duplicated from line 44

Reply:
Oh yes, it is. Let me remove it.

Reply:
Fixed, thanks.